src/sdk/pynni/tests/test_classic_nas.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import glob
+import json
+import logging
+import os
+import random
+import shutil
+import sys
+from collections import deque
+from unittest import TestCase, main
+
+from nni.tuner import Tuner
+from nni.regularized_evolution_tuner.regularized_evolution_tuner import RegularizedEvolutionTuner
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger('test_tuner')
+
+
+class ClassicNasTestCase(TestCase):
```

Comment:
Is it necessary that we copy the code?

Reply:
Maybe you mean the code should be integrated into `test_builtin_tuner.py`.
I think the search space for NAS varies from HPO, so integrating the code can be misleading when others trying to add ut for classic nas tuner.